mgmtsystem_quality/__manifest__.py

```diff
@@ -3,20 +3,18 @@
 {
     "name": "Quality Management System",
     "summary": "Manage your quality management system",
-    "version": "11.0.1.0.0",
-    "author": "Savoir-faire Linux, "
-              "Odoo Community Association (OCA)",
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux, " "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/management-system",
     "license": "AGPL-3",
     "category": "Management System",
     "depends": [
+        "mgmtsystem_manual",
```

Comment:
Why do we want to make this module mandatory?
Is it really required?

Reply:
This module create manual_id field on mgmtsystem.system model. In the roadmap this repo will be multi system and this document is necesary. [+INFO](https://odoo-community.org/groups/management-systems-17/management-systems-158545?mode=thread&date_begin=&date_end=)